parm/varmap_tables/README

```diff
@@ -14,7 +14,8 @@ Column 2: Name the code will use to save the variable in the output file. Unimpl
 Comumn 3: Behavior when the code can't find the variable in the input file. Options are:
 	  "skip": Don't write to output file.
 	  "set_to_fill": Set to user-specified field value (see column 4).
-	  "intrp": LnP interpolation to missing levels. No extrapolation allowd.
+	  "intrp": Ln(pressure) interpolation to missing levels. Extrapolation is allowed, 
+                 but should be used with caution.
 	  "stop": Force an exception and stop code execution. Use this if you absolutely
```

Comment:
The formatting looks strange here, but it's OK in the README file when viewed in vi.